seatunnel-translation/seatunnel-translation-spark/seatunnel-translation-spark-3.3/src/main/java/org/apache/seatunnel/translation/spark/serialization/InternalRowConverter.java

```diff
@@ -241,4 +244,26 @@ private static Object reconvertArray(ArrayData arrayData, ArrayType<?, ?> arrayT
         }
         return newArray;
     }
+
+    public Object[] convertToFields(InternalRow internalRow, StructType structType) {
```

Comment:
if the method not support array or map, it will cause exception when handler array / map

Reply:
For example when  I query a row from source, but I just want convert it's name it will exception;

Reply:
ok, I has fixed it